src/pages/networks/forms/NetworkForm.tsx

```diff
@@ -0,0 +1,713 @@
+                  options={optionYesNo}
+                />
+              )}
+              {formik.values.type !== "ovn" &&
+                formik.values.ipv6_address !== "none" && (
+                  <Select
+                    {...getFormProps("ipv6_firewall")}
+                    label="IPv6 firewall"
+                    help="Whether to generate filtering firewall rules for this network"
+                    options={optionYesNo}
+                  />
+                )}
+              {formik.values.ipv6_nat === "true" && (
+                <>
+                  <Input
+                    {...getFormProps("ipv6_nat_address")}
+                    type="text"
+                    label="IPv6 NAT address"
+                    help="The source address used for outbound traffic from the bridge"
+                  />
+                  {formik.values.type !== "ovn" && (
+                    <Select
+                      {...getFormProps("ipv6_nat_order")}
+                      label="IPv6 NAT order"
+                      help="Whether to add the required NAT rules before or after any pre-existing rules"
+                      options={[
+                        {
+                          label: "Select option",
+                          value: "",
+                          disabled: true,
+                        },
+                        {
+                          label: "Before",
+                          value: "before",
+                        },
+                        {
+                          label: "After",
+                          value: "after",
+                        },
+                      ]}
+                    />
+                  )}
+                </>
+              )}
+              {formik.values.type !== "ovn" &&
+                formik.values.ipv6_address === "none" && (
+                  <>
+                    <Input
+                      {...getFormProps("ipv6_ovn_ranges")}
+                      type="text"
+                      label="IPv6 ovn ranges"
+                      help="Comma-separated list of IPv6 ranges to use for child OVN network routers (FIRST-LAST format)"
+                    />
+                    <Input
+                      {...getFormProps("ipv6_ovn_routes")}
+                      type="text"
+                      label="IPv6 ovn routes"
+                      help="Comma-separated list of additional IPv6 CIDR subnets to route to the bridge"
+                    />
+                    <Select
+                      {...getFormProps("ipv6_ovn_routing")}
+                      label="IPv6 ovn routing"
+                      options={optionYesNo}
+                      help="Whether to route traffic in and out of the bridge"
+                    />
+                  </>
+                )}
+            </>
+          )}
+          {[USER, ADVANCED_OVN].includes(section) && (
+            <>
+              <Label>User-provided free-form key/value pairs</Label>
```

Comment:
This "User" section should be removed now that we have added the YAML configuration section. Allowing the user to add free-form key-value pairs is not specific to networks. It's a feature common with instances, profiles, and whatnot. So I don't see why this should be here considering that we don't have it for instances and profiles, and also that with the YAML configuration section you can allow the user to achieve exactly the same result.